### PR TITLE
Unit Tests

### PR DESCRIPTION
**Pull request by daria-github**
* Added final unit tests for all current utils 
* Updated tests for a couple of edge cases

### CODE REVIEW

File: helpers/string.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { Conversation } from '@xmtp/xmtp-js';
 import { NextRouter } from 'next/router';
 
-export const truncate = (str: string | undefined, length: number): string | undefined => {
+export const truncate = (str: string | undefined, length: number): string => {
   if (!str) {
-    return str;
+    return '';
   }
   if (str.length > length) {
```
Comment (bhavya2611) [inline helpers/string.ts:8]: should this be a else if

**Review by bhavya2611 — APPROVED** [review]
LG - minor comment
